docs/tutorial.rst

```diff
@@ -178,6 +204,39 @@ If you run ``nox --list-sessions``, you'll see that this generates the following
 
 If you only want to run one of the parametrized sessions, see :ref:`running_paramed_sessions`.
 
+Giving friendly names to parametrized sessions
+----------------------------------------------
+
+The automatically generated names for parametrized sessions, such as ``tests(django='1.9', database='postgres')``, can be long and unwieldy to work with even with using :ref:`keyword filtering <opt-sessions-and-keywords>`. You can give parametrized sessions custom IDs to help in this scenario. These two examples are equivalent:
+
+::
+
+    @nox.session
+    @nox.parametrize('django',
+        ['1.9', '2.0'],
+        ids=['old', 'new'])
+    def tests(session, django):
+        ...
+
+::
+
+    @nox.session
+    @nox.parametrize('django', [
+        nox.param('1.9', id='old'),
+        nox.param('2.0', id='new'),
+    ])
+    def tests(session, django):
+        ...
+
+When running ``nox --list`` you'll see their new IDs:
+
+.. code-block:: console
+
+    * tests(old)
+    * tests(new)
+
+And you can run them with ``nox --sessions "tests(old)"`` and so on.
```

Comment:
This example leaves me wanting to know what happens when there are two or more parameters as your `tests(django='1.9', database='postgres')` example hints at